### PR TITLE
Added pyfhel hello world demo nb

### PR DESCRIPTION
**Pull request by suppathak**
Added a notebook exploring the FHE schemes "bfv" and "ckks" along with examples by taking and integer and float variables. 

Related : #11

### CODE REVIEW

File: notebooks/pyfhel_hello_world.ipynb [hunk 1]
```diff
@@ -0,0 +1,1086 @@
+{
```
[the remark this answers is not in the document]
Comment (suppathak) [reply notebooks/pyfhel_hello_world.ipynb:1]: Thanks @MichaelClifford . I will try to add these points to the notebook.

**Review by erikerlandson — APPROVED** [review]

**Comment by erikerlandson** [comment]
This is great!  I notice a lack of division, at least dividing one encrypted value by another encrypted value

**Comment by suppathak** [comment]
> This is great! I notice a lack of division, at least dividing one encrypted value by another encrypted value

Thank you @erikerlandson . Have looked into it. However, Pyfhel package does not support any division.  One way thorough would be to find the clear text inverse and then multiplying with the ciphertext inverse..

**Comment by erikerlandson** [comment]
I see you can do:   encrypted / unencrypted, like with the average example:  `(e1 + e2) / 2`.

Out of curiosity, does:  `1 / e` work?  Because if that works one could do:   `d = 1 / e`:  followed by `e2 * d`

**Comment by suppathak** [comment]
> Out of curiosity, does: `1 / e` work? Because if that works one could do: `d = 1 / e`: followed by `e2 * d`

Unfortunately, no. Not for the ciphertext values and not even for plain text values. The only way is, `1/(actual_value)`, and then followed by encryption and multiplication is the way out.

**Comment by erikerlandson** [comment]
One other I forgot:  I see they provide `e1 == e2` (which is of course easy) - can they do `e1 < e2` ?

**Comment by suppathak** [comment]
> I see they provide `e1 == e2` (which is of course easy) - can they do `e1 < e2` ?

This is a good point. I will try to update these point in another PR. 
The point is Pyfhel only offers, `==` and `!=` comparison. It does not support `>` or `<` comparison operations. One of the reason is for the security of cipher-text itself. Any cipher we would trust will generate a cipher text that looks random, which is one of the criteria we use on whether the cipher is trustworthy. If it is not random, it isn't.

We will however try to look into it more, to see if there is any clever mathematics that goes into comparing two ciphertexts.

**Comment by erikerlandson** [comment]
It may be that `<` is unsafe - because if you have that, you can narrow your value down to any precision by trial and error, which (as you say) would defeat the value of the encryption.

**Pull request merged**